summoner-cli/src/Summoner/Template/GitHub.hs

```diff
@@ -250,22 +244,20 @@ gitHubFiles Settings{..} = concat
 
           - |
             if [ -z "$$STACK_YAML" ]; then
-              ghc --version
-              cabal --version
-              cabal new-update
-              cabal new-build --enable-tests --enable-benchmarks
+              $cabalUpdate
+              $cabalBuild
             else
               curl -sSL https://get.haskellstack.org/ | sh
               stack --version
-              stack build --system-ghc --test --bench --no-run-tests --no-run-benchmarks --ghc-options=-Werror
+              $stackBuild
             fi
 
         script:
           - |
             if [ -z "$$STACK_YAML" ]; then
                ${cabalTest}
```

Comment:
I just noticed that we have 3 spaces here. Do you think we need to change to 2 for consistency?